test/integration/resize_test.go

```diff
@@ -32,9 +33,13 @@ var _ = Describe("vary VM parameters: memory cpus, disk", Serial, Ordered, Label
 		})
 
 		It("start CRC", func() {
-			Expect(
-				crcSuccess("start", "--memory", "12000", "--cpus", "5", "--disk-size", "40", "-p", pullSecretPath)).
-				To(ContainSubstring("Started the OpenShift cluster"))
+			start := time.Now()
+			message := crcSuccess("start", "--memory", "12000", "--cpus", "5", "--disk-size", "40", "-p", pullSecretPath)
+			duration := time.Since(start)
+			Expect(message).To(ContainSubstring("Started the OpenShift cluster"))
+
+			data := "crc start(default): " + duration.String() + "\n"
+			writeDataToFile("time-consume.txt", data)
```

Comment:
This file `time-consume.txt` is created in the current executing test directory, can we put that something like https://github.com/crc-org/crc/blob/main/test/e2e/testsuite/testsuite.go#L581 one so that in case of failure of success it become part of test result for CI? 
 https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/crc-org_crc/4809/pull-ci-crc-org-crc-main-e2e-crc/1939943461000056832/artifacts/e2e-crc/gather-crc/artifacts/out/test-results/ (like this)